create_icr_file/create_icr_file.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import os
+import random
+
+from core_data_modules.traced_data.io import TracedDataJsonIO, TracedDataCSVIO
+from core_data_modules.util import IOUtils
+
+# Number of messages to export in the ICR file
+ICR_MESSAGES_COUNT = 200
+# Number of characters below which messages are considered noise
+NUMBER_OF_CHARS_NOISE = 15
+
+if __name__ == "__main__":
```

Comment:
Why is this file formatted with such short lines?

Reply:
Pep8 only allows for line 79 characters long and I find it more readable that way

Reply:
Ah, so the one exception we had been making to PEP-8 was on line length. The rest of AVF is, for the most part, limited to 120 chars.